### PR TITLE
Adds sit_cma_layer to config.yml for SIT deployments

### PR DESCRIPTION
**Pull request by flamingbear**
**Summary:**

Fixes for SIT
Updates shared log location for SIT environment
Updates configuration to specify CMA Lambda layer by environemnt. 

Details:
We can't share lambda layers yet across accounts so need to have separate
versions for the cma_layer in each environment.
this adds a `sandbox_cma_layer` and a `sit_cma_layer` and sets the default `cma_layer` to sandbox.

Then we override the default cma_layer in SIT deployment stacks with `sit_cma_layer`.

**NOTE**: Bamboo is currently configured to run this branch on SIT since that is where the changes are being made.  the environment variable NGAP_ENV is set to SIT. so the tests are running on the default cumulus stack `cumulus-sit` .

### CODE REVIEW

File: example/app/config.yml [hunk 1]
```diff
@@ -61,7 +61,9 @@ default:
   # test configuration ends
 
   cmaDir: '/opt/'
-  cma_layer: arn:aws:lambda:us-east-1:{{AWS_ACCOUNT_ID}}:layer:Cumulus_Message_Adapter:4
+  sandbox_cma_layer: arn:aws:lambda:us-east-1:{{AWS_ACCOUNT_ID}}:layer:Cumulus_Message_Adapter:4
+  sit_cma_layer: arn:aws:lambda:us-east-1:{{AWS_ACCOUNT_ID}}:layer:Cumulus_Message_Adapter:1
+  cma_layer: '{{sandbox_cma_layer}}'
```
Comment (flamingbear) [inline example/app/config.yml]: Because we can't share lambda layers across accounts we have to have separate ones for each account.

File: example/app/config.yml [hunk 2]
```diff
@@ -736,6 +741,46 @@ mhs4:
...
+  apiConfigs:
+    backend:
+      private: true
+      port: 8022
+    distribution:
+      private: true
+      port: 7022
+      logApiGatewayToCloudWatch: true
+  logToSharedDestination: '{{SHARED_LOG_DESTINATION_ARN}}'
+  cma_layer: '{{sit_cma_layer}}'
```
Comment (markdboyd) [inline example/app/config.yml]: So this deployment is for SIT?

File: bamboo/set-bamboo-env-variables.sh [hunk 3]
```diff
@@ -114,6 +114,7 @@ if [[ $bamboo_NGAP_ENV = "SIT" ]]; then
   export VPC_CIDR_IP=$bamboo_SECRET_SIT_VPC_CIDR_IP
   export PROVIDER_HOST=$bamboo_SECRET_SIT_PROVIDER_HOST
   export SECURITY_GROUP=$bamboo_SECRET_SIT_SECURITY_GROUP
+  export SHARED_LOG_DESTINATION_ARN=$bamboo_SIT_SHARED_LOG_DESTINATION_ARN
```
Comment (markdboyd) [inline bamboo/set-bamboo-env-variables.sh]: Unrelated?

File: example/app/config.yml (hunk 2 above)
Comment (flamingbear) [reply example/app/config.yml]: Yeah mhs2 is just my personal SIT deployment, and I updated it for Lauren and Jon after asking if they wanted me to.

File: bamboo/set-bamboo-env-variables.sh (hunk 3 above)
Comment (flamingbear) [reply bamboo/set-bamboo-env-variables.sh]: related to fixing SIT yes, because we have to have different endpoints for the shared log destination based on the metrics account ID.
